src/lib/crypt.test.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @jest-environment node
+ */
+
+import { hash } from './crypt';
+
+describe('Crypt', () => {
+  describe('hash()', () => {
+    it('should generate hash for the string', async () => {
```

Comment:
Avoid the old syntax `should`, prefer the imperative tone:

```suggestion
    it('generates a hash for the string', async () => {
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/44/commits/772649044e477fe0425328fbaae9c5acd0eccb73